mitodl/ocw-studio
# Added WebsiteStarter model with local development support

#### Pre-Flight checklist


#### What are the relevant tickets?
Partially addresses #37 

#### What's this PR do?
- Adds the `WebsiteStarter` model to (a) track the available starter projects that an author can use to create their site and (b) store their site config
- Allows developers to define starter projects in the local filesystem (instead of fetching them from Github) and load them into the database via management command

#### How should this be manually tested?
Run the `populate_local_starters` management command and check out the Django admin class

#### Any background context you want to provide?
The structure of the site config is still TBD, so the contents of `localdev/starters/example-starter/ocw-studio.yml` are likely to change in the future. This is just a rough draft.
